<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Danger</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>707</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OctoShot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/octotools/octo-tools.svg</normaloff>:/icons/octotools/octo-tools.svg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">padding:5px; color:#999999; margin-top:0px; margin-bottom:0px; border:0px solid #111111; border-radius:5px; background-color:#333333; font-size: 22px</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="statusTip">
    <string>Node Graph</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1d1d1d; color:#3d3d3d;padding:10px; border:1px dashed #333;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>[ Node Graph ]</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="Status">
   <property name="styleSheet">
    <string notr="true">padding:10px</string>
   </property>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="statusTip">
    <string>Tool Bar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:10px;</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="cursor">
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="windowIcon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/outlined/Editor/table_chart.svg</normaloff>:/icons/svg/material/outlined/Editor/table_chart.svg</iconset>
   </property>
   <property name="statusTip">
    <string>Object Tree</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:10px; border:1px dashed #333;</string>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#1d1d1d; color:#3d3d3d;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>[ Object Tree ]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="cursor">
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="statusTip">
    <string>Node Parameters</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:10px; border:1px dashed #333;</string>
   </property>
   <property name="windowTitle">
    <string>Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#1d1d1d; color:#3d3d3d;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>[ Node Parameters ]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="cursor">
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="statusTip">
    <string>Output Log</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:10px; border:1px dashed #333;</string>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#1d1d1d; color:#3d3d3d;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>[ Logging Output ]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="cursor">
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="statusTip">
    <string>Data Spreadsheet</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:10px; border:1px dashed #333;</string>
   </property>
   <property name="windowTitle">
    <string extracomment="choice as">Data </string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#1d1d1d; color:#3d3d3d;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>[ Data Spreadsheet ]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="RightToolBar">
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>46</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_2"/>
    <addaction name="actionOpen_3"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/octotools/task.svg</normaloff>:/icons/octotools/task.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/octotools/folder_02.svg</normaloff>:/icons/octotools/folder_02.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/octotools/floppy-disk.svg</normaloff>:/icons/octotools/floppy-disk.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/outlined/Content/save.svg</normaloff>:/icons/svg/material/outlined/Content/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="statusTip">
    <string>Save File As...</string>
   </property>
  </action>
  <action name="actionRecent_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/TwoTone/Image/timelapse.svg</normaloff>:/icons/svg/material/TwoTone/Image/timelapse.svg</iconset>
   </property>
   <property name="text">
    <string>Recent File 01</string>
   </property>
  </action>
  <action name="actionRecent_File_02">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/TwoTone/Image/timelapse.svg</normaloff>:/icons/svg/material/TwoTone/Image/timelapse.svg</iconset>
   </property>
   <property name="text">
    <string>Recent File 02</string>
   </property>
  </action>
  <action name="actionRecent_File_03">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/TwoTone/Image/timelapse.svg</normaloff>:/icons/svg/material/TwoTone/Image/timelapse.svg</iconset>
   </property>
   <property name="text">
    <string>Recent File 03</string>
   </property>
  </action>
  <action name="actionMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/outlined/Toggle/toggle_off.svg</normaloff>
     <normalon>:/icons/svg/material/outlined/Toggle/toggle_on.svg</normalon>:/icons/svg/material/outlined/Toggle/toggle_off.svg</iconset>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="toolTip">
    <string>Mode</string>
   </property>
   <property name="statusTip">
    <string>View Mode Switch</string>
   </property>
  </action>
  <action name="actionObjects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/TwoTone/Content/font_download.svg</normaloff>:/icons/svg/material/TwoTone/Content/font_download.svg</iconset>
   </property>
   <property name="text">
    <string>Objects</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </action>
  <action name="actionData_Spreadsheet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Data Spreadsheet</string>
   </property>
  </action>
  <action name="actionOutput_Log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Output Log</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
   <property name="toolTip">
    <string>File</string>
   </property>
  </action>
  <action name="actionlogo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/octotools/octo-tools.svg</normaloff>:/icons/octotools/octo-tools.svg</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>File</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>Window</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/outlined/Image/crop_din.svg</normaloff>:/icons/svg/material/outlined/Image/crop_din.svg</iconset>
   </property>
   <property name="text">
    <string>Maximize</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/TwoTone/Content/remove.svg</normaloff>:/icons/svg/material/TwoTone/Content/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/svg/material/TwoTone/Navigation/close.svg</normaloff>:/icons/svg/material/TwoTone/Navigation/close.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_3">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
